TestPlaybooks/playbook-CreateIndicatorFromSTIXTest.yml

```diff
@@ -186,13 +186,13 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quiet: false
-  "5":
-    id: "5"
-    taskid: "6"
+  "deletecontextonstix":
```

Comment:
you shouldnt change the ids of the PlaybookTask only the taskid and the inside the task
![image](https://user-images.githubusercontent.com/7270217/73607187-9ccb6e00-45bb-11ea-979a-be30dcb0b9ef.png)

Reply:
it's working, it's only for the testplaybook.
let's leave it that way and merge.